pkg/diag/validator/pod_test.go

```diff
@@ -598,6 +598,45 @@ func TestRun(t *testing.T) {
 					ErrCode: proto.StatusCode_STATUSCHECK_CONTAINER_RESTARTING,
 				}, []string{"[foo foo-container] some panic"})},
 		},
+		{
+			description: "pod condition with events when pod Initilizing phase",
```

Comment:
```suggestion
			description: "pod condition with events when pod is in Initializing phase",
```

Reply:
ah i just switched to VSC and I need to install spellcheck plugin